src/Resource/Users.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Fschmtt\Keycloak\Resource;
+
+use Fschmtt\Keycloak\Collection\UserCollection;
+use Fschmtt\Keycloak\Http\Command;
+use Fschmtt\Keycloak\Http\Method;
+use Fschmtt\Keycloak\Http\Query;
+use Fschmtt\Keycloak\Representation\User as UserRepresentation;
+
+class Users extends Resource
```

Comment:
I know it requires some effort, but could you add unit and integration tests for the new resource?

If you're not able to, please let me know and I'll be happy to help you out!